Harvest.Web/ClientApp/src/Shared/ProjectHeader.tsx

```diff
@@ -7,6 +9,25 @@ interface Props {
 
 export const ProjectHeader = (props: Props) => {
   const { project, title } = props;
+  const [modal, setModal] = useState<boolean>(false);
+  const toggleModal = () => setModal(!modal);
```

Comment:
whenever you set state based on existing state, you should use the callback to ensure you are working with the proper value.  So here it should be `const toggleModal = () => setModal(m => !m);`